src/main/scala/exercises/04_Options.scala

```diff
@@ -34,7 +34,25 @@ object Options {
    * - does not exist: 					"not existing"
    */
   def roomState(rooms: Map[Int, Option[String]], room: Int): String = {
-    error("Fix me")
+    if (!rooms.contains(room)) {
```

Comment:
I did it differently, using .get and pattern matching all the way through. I tried to look online for which was preferred because I had to use `case Some(Some("locked"))` which seems less elegant than `case Some("locked")` but did not use an if statement, but I could not find anything. It would be interesting to look more into that.